core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -550,9 +550,16 @@ oppia.factory('explorationRightsService', [
         var that = this;
 
         requestParams.version = explorationData.data.version;
-        var explorationRightsUrl = (
-          '/createhandler/rights/' + explorationData.explorationId);
-        $http.put(explorationRightsUrl, requestParams).then(function(response) {
+        var requestUrl = '';
```

Comment:
Suppose, If I make a function like:
statusChange(make_public, make_publicized) {}
Won't it force others to call it with both arguments every time?

I moved on to implementing two functions, savePublishStatusChanges(make_public) and savePublicizeStatusChange(make_publicized).

But if I follow same line of argument it will introduce a lot of new functions ?